### DISCUSSION TITLE
Switching types of trees with a hotkey before building it?

### DISCUSSION THREAD

**Discussion by mathmaj** [Keriew/augustus#312] [Ideas]
Good evening.

I think it would be really great if we could switch tree type to another before building it with a hotkey. For example, the same hotkey as for changing the orientation of a building.

I often want to place several tree varieties at random. But returning each time to the menu with the mouse is laborious. 

I like the idea of the hotkey, but if it is not possible, maybe we could make an "all" button like for temples .

Thanks for reading!

**Comment by Keriew** [comment]
That's a good idea. We've added a system for cycling similar buildings by using the rotation key for the next release, I've added that functionality for trees in 19daad8.

**Reply by mathmaj** [reply]
This will be awesome!! Thanks :)
